share/doc/homebrew/Acceptable-Formulae.md

```diff
@@ -57,9 +57,10 @@ point it to the downloaded archive in order to avoid loading.
 
 ### We don’t like binary formulae
 Our policy is that formulae in the core repository
-([homebrew/core](https://github.com/Homebrew/homebrew-core)) must be built
-from source (or produce cross-platform binaries like e.g. Java). Binary-only
-formulae should go to [homebrew/binary](https://github.com/Homebrew/homebrew-binary) or [homebrew-cask](https://github.com/caskroom/homebrew-cask).
+([homebrew/core](https://github.com/Homebrew/homebrew-core)) must be open-source
+and either built from source or produce cross-platform binaries like e.g. Java).
+Binary-only formulae should go to
+[homebrew-cask](https://github.com/caskroom/homebrew-cask).
```

Comment:
I wonder if this should be “Homebrew Cask” instead of “homebrew-cask” as the link text, as the latter is very close to how we otherwise refer to our taps (except for the hyphen instead of the slash) and could cause more confusion. Thoughts?